exporters/elasticsearch/BUILD

```diff
@@ -0,0 +1,40 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "es_log_exporter",
+    srcs = [
+        "src/es_log_exporter.cc",
+    ],
+    hdrs = [
+        "include/opentelemetry/exporters/elasticsearch/es_log_exporter.h",
+    ],
+    strip_include_prefix = "include",
+    deps = [
+        "//ext:headers",
+        "//sdk/src/logs",
+        "@curl",
+        "@github_nlohmann_json//:json",
+    ],
+)
+
+cc_test(
+    name = "es_log_exporter_test",
+    srcs = ["test/es_log_exporter_test.cc"],
+    copts = [
+        "-DCURL_STATICLIB",
+    ],
+    linkopts = select({
+        "//bazel:windows": [
+            "-DEFAULTLIB:advapi32.lib",
+            "-DEFAULTLIB:crypt32.lib",
+            "-DEFAULTLIB:Normaliz.lib",
+        ],
+        "//conditions:default": [],
+    }),
```

Comment:
Since you're baking the 'curl' client into the exporter itself, you need the copts to be part of the library above.

Reply:
Worked like a charm, thanks Josh :)